lib/semantics/check-omp-structure.cc

```diff
@@ -246,7 +294,47 @@ void OmpStructureChecker::Enter(const parser::OmpProcBindClause &) {
 void OmpStructureChecker::Enter(const parser::OmpReductionClause &) {
   CheckAllowed(OmpClause::REDUCTION);
 }
-void OmpStructureChecker::Enter(const parser::OmpScheduleClause &) {
+
+bool OmpStructureChecker::ScheduleModifierHasType(
+    const parser::OmpScheduleClause &x,
+    const parser::OmpScheduleModifierType::ModType &type) {
+  const auto &modifier{std::get<0>(x.t)};
+  if (modifier.has_value()) {
+    const auto &modType1{std::get<0>(modifier->t)};
+    const auto &modType2{std::get<1>(modifier->t)};
```

Comment:
I think you should have explicit types here to improve readability. Either replace the number or auto with the real type. Otherwise you have to look at `OmpScheduleClause` to know what any of these types are.

Reply:
Addressed in commit [1b57193](https://github.com/flang-compiler/f18/pull/536/commits/1b57193d2377c5121f16075950b64ea77fa72e2e).